kerykeion/primary_directions/directions_factory.py

```
@@ -92,105 +123,123 @@ def compute(
+                            )
+                            # OA/OD of the aspect point under the significator's pole
+                            if sig_is_eastern:
+                                oa_prom = PrimaryDirectionsFactory._oblique_ascension(
+                                    ra_asp, dec_asp, sig.pole
+                                )
+                            else:
+                                oa_prom = PrimaryDirectionsFactory._oblique_descension(
+                                    ra_asp, dec_asp, sig.pole
+                                )
+                        except Exception:
+                            continue
```

Ruff (0.15.15)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)